virttest/qemu_vm.py

```diff
@@ -4315,6 +4315,36 @@ def check_block_locked(self, value):
                     return block['locked']
         return False
 
+    def check_device_tray_opened(self, value):
```

Comment:
Well, I think we should not continue to add the `check_something` method into `VM` class to make this file longer. Instead, a better structure would be:
```python
tray_status = vm.devices.get_node_by_path("/machine/cdrom1").tray_status
test.assertTrue(tray_status == "opened", ERROR_MESSAGE)
```
Currently, we do not have `vm.devices.get_node_by_path()` and `CdromDev.tray_status`, and it is not easy to implement, so as a trade-off, we can implement it as a function and put it into tp-qemu, like https://github.com/autotest/tp-qemu/pull/1388